packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/extensions/DevMenuExtension.kt

```diff
@@ -83,15 +83,6 @@ class DevMenuExtension(reactContext: ReactApplicationContext) :
 
     if (devSettings is DevInternalSettings) {
       action("js-inspector", devDelegate::openJsInspector) {
-        isAvailable = {
```

Comment:
can you clarify why we're removing this boolean?

Reply:
it looks like it was moved to `isJSInspectorAvailable` in DevMenuDevSettings: https://github.com/expo/expo/pull/17762/files#diff-5fa64fd47cec5f6f3ebff98d627a4e7b7e78a2f615e6b5c5ec70b19f84c7415aR26

Reply:
It is not used here and was moved to the  `DevMenuDevSettings`